env-config/example.env

```diff
@@ -27,14 +23,17 @@ PASSWORD=
 CLOUD_SQL_CONNECTION_NAME=
 RUBY_BACKEND=
 FLASK_ENV=
+FLASK_LOCAL_PORT=8080 # only needed in local.env
 
 EXPRESS_APP_ENGINE_SERVICE=
 EXPRESS_SENTRY_PROJECT=
-EXPRESS_PORT=
+EXPRESS_LOCAL_PORT=
 EXPRESS_ENV=
+EXPRESS_LOCAL_PORT=8088 # only needed in local.env
 
 RUBY_APP_ENGINE_SERVICE=
 RUBY_SENTRY_PROJECT=
+RUBY_LOCAL_PORT=4567 # only needed in local.env
```

Comment:
Is there nothing we have to do with our current Ruby setup when changing this variable?

Reply:
Good question. I didn't see any port specified anywhere in our Ruby code explicitly.
When I tried running it, looks like it's using that port by default:

```
./run.sh
[2022-12-19 16:10:47] INFO  WEBrick 1.4.4
[2022-12-19 16:10:47] INFO  ruby 2.6.10 (2022-04-12) [universal.arm64e-darwin21]
== Sinatra (v2.2.0) has taken the stage on 4567 for development with backup from WEBrick
[2022-12-19 16:10:47] INFO  WEBrick::HTTPServer#start: pid=14821 port=4567
```